addons/mrp/models/mrp_bom.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import openerp.addons.decimal_precision as dp
+from openerp import api, fields, models
+from openerp.tools.translate import _
+from openerp.exceptions import UserError
+from openerp.tools import float_round
```

Comment:
From whole import use odoo instead of openerp, if i am wrong please correct me :turtle: